apps/web/src/pages/index.tsx

```diff
@@ -52,12 +53,21 @@ const Home: NextPage<{ crowdfunds: MatchingPool[] }> = ({
   );
 };
 
+
 export async function getServerSideProps(ctx: GetServerSidePropsContext) {
+  const oldCrowdfundsIds = [
+  '0x2d26e03ada901193970e1aa193b7d9fed736726b',
+   '0x74ad514636f1386f374e1107435f35d393d0b279',
+   '0xed6a325127b7347715624d7b238ad0347dd04e48',
+   '0xfd87a966294e29af331842b6a1426c541a41d9b6']
+   
```

Comment:
I think you should be able to query based on `createdAt`.

Maybe something like this:
```gql
query getCrowdfunds($first: Int, $skip: Int, where: { createdAt_gt: <timestamp in seconds> } ) {
  ...
}
``` 

By block might also work.

- https://github.com/supermodularxyz/crowdfund-matchingpool/blob/main/web3/subgraph/src/schema.graphql#L11
- https://github.com/supermodularxyz/crowdfund-matchingpool/blob/main/apps/web/src/hooks/useCrowdfund.ts#L34-L40
- https://thegraph.com/docs/en/querying/graphql-api/#example-for-block-filtering